src/actions/enter.ts

```diff
@@ -150,6 +152,16 @@ export class EnterAction {
     // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
     workspace.getCursor()?.setCurNode(ASTNode.createBlockNode(newBlock)!);
     this.mover.startMove(workspace);
+
+    const isTopLevelBlock =
```

Comment:
Nit: `isStartBlock` or the like might be clearer here; any block can be a top-level block, but this appears to be looking for start/"hat" blocks specifically?

Reply:
Thanks, 40927bb6866cf8f5f0c2c186d157d211111cc44e